cypress/integration/albumCards.js

```diff
@@ -1,48 +1,59 @@
 context('Album Cards', () => {
   beforeEach(() => {
     cy
+    .intercept('POST', 'https://pure-hollows-05817.herokuapp.com/https://tranquil-depths-91575.herokuapp.com/graphql', (req) => {
+      if (req.body.query.includes('The Payback')) {
+        req.reply({ fixture: 'details-fixture.json' })
+      }
+    })
     .intercept('POST', 'https://pure-hollows-05817.herokuapp.com/https://tranquil-depths-91575.herokuapp.com/graphql', (req) => {
```

Comment:
Do you think we will still need the proxy address for these intercepts?

Reply:
we do not